src/track/cue.cpp

```diff
@@ -11,7 +11,21 @@
 #include "util/color/color.h"
 
 namespace {
-    const QString kDefaultLabel = ""; // empty string, not null
+
+const QString kDefaultLabel = ""; // empty string, not null
```

Comment:
I'm not sure, would `QStringLiteral` make a difference here?

Reply:
Just remove  = "";
Or use = QString();

Reply:
I have only reformatted the code.

kDefaultLabel becomes obsolete with QString() which is implicitly optional. I'm not sure what is intended here?